sunpy/net/dataretriever/tests/test_lyra_ud.py

```diff
@@ -15,6 +21,97 @@
 LCClient = lyra.LYRAClient()
 
 
+def mock_query_object(start_date, end_date):
+    """
+    Creation of a QueryResponse object, and prefill some
+    downloaded data from lyra.LYRAClient().fetch(Time('20 ..)
+    """
+    # Create a mock QueryResponse object
+    map_ = {
+        'TimeRange': TimeRange(parse_time(start_date), parse_time(end_date)),
```

Comment:
`TimeRange(start_date, end_date)` should do if I remember correctly. If it doesn't work feel free to ignore.